### DISCUSSION TITLE
Is this wild glob a bug?

### DISCUSSION THREAD

**Discussion by StLeoX** [eza-community/eza#818] [Q&A, Howto, Help]
For this glob `./*.bkp`, I'm not sure if the output is right.  
I thought `*` would catch `.$a`.

![exa bug 2，eza 有类似行为](https://github.com/eza-community/eza/assets/59193798/68adb238-e7fb-4866-8e70-9c3a9e3a2cae)

Here is the `help wildcards-globbing` result:

![fish glob](https://github.com/eza-community/eza/assets/59193798/2611c355-14c8-4a6f-b7a9-39d0310b8ccd)

**Comment by cafkafk** [comment]
I'm unsure if this is a question related to eza, rather this seems to be a question of how fish handles globbing.

E.g. it also affects ls:
```
✦ ❯ ls -a ./*.bkp
fish: No matches for wildcard './*.bkp'. See `help wildcards-globbing`.
ls -a ./*.bkp
      ^~~~~~^
```

Consider asking in the fish-shell matrix room, they're usually quite nice https://matrix.to/#/#fish-shell:matrix.org :)

If I'm missing something, please feel free to reply and I'll reopen.

**Reply by StLeoX** [reply]
You can switch to bash. And it will be the same testing output.

**Reply by StLeoX** [reply]
So, if this behavior is compatible with `ls`, then it's right?
